### ISSUE TITLE
Completely broken in Pancake v4

### ISSUE THREAD

**Issue by sbufe** [zerosignalproductions/Freshness-Pancake-Theme#12]
It displays the header with the horizontal menu beginning at Dashboard and Invoices, and then it's blank for the rest of the menu, and no content below that.

To fix (without being able to access settings pages), have to go into phpMyAdmin and edit the pancake_settings table, admin_theme value, manually back to a valid theme.

**Comment by nordurljosahvida** [comment]
I noticed you can also simply delete the two zerosignal foders from the themes directory and Pancake will revert to the default theme. I wrote zerosignal: hoping in some kind of feedback. Any idea as to how to manually fix it for version 4 otherwise?

**Comment by sbufe** [comment]
No, I don't.  A simple fix that would get it running in v4 would be very welcome.
